src/packages/@ncigdc/modern_components/CohortComparison/CohortComparison.js

```diff
@@ -169,18 +163,16 @@ export default compose(
             palette={[set1_colour, set2_colour]}
           />
```

Comment:
probably not for this ticket but Survival table doesn't have S1/S2.

Reply:
It does doesn't it? Do you mean the ones on the graph itself?

Reply:
![image](https://user-images.githubusercontent.com/167161/30454367-0fdb04d8-996a-11e7-80c7-98bce16af9e6.png)

Reply:
Oh you mean on the table? I can do that here.